### DISCUSSION TITLE
Any way to add "copy to clipboard" button to code blocks?

### DISCUSSION THREAD

**Discussion by duncsand** [gonzalezreal/swift-markdown-ui#361] [Q&A] [answered]
Just this. When rendering markdown there may be code blocks and I'd like to add a "copy to clipboard" button to each code block. Is there a way to do this?

**Comment by gonzalezreal** [comment, marked as answer]
Hi @duncsand,

The [Demo app](https://github.com/gonzalezreal/swift-markdown-ui/blob/main/Examples/Demo) shows how to do this:
https://github.com/gonzalezreal/swift-markdown-ui/blob/a9c7615fb50323069c2979c69263973aa1b24a8f/Examples/Demo/Demo/CodeSyntaxHighlightView.swift#L5
